diff --git a/node_modules/scratch-vm/src/extension-support/extension-manager.js b/node_modules/scratch-vm/src/extension-support/extension-manager.js
index a5e2c4e..2312c75 100644
--- a/node_modules/scratch-vm/src/extension-support/extension-manager.js
+++ b/node_modules/scratch-vm/src/extension-support/extension-manager.js
@@ -22,6 +22,10 @@ const builtinExtensions = {
     videoSensing: () => require('../extensions/scratch3_video_sensing'),
     ev3: () => require('../extensions/scratch3_ev3'),
     makeymakey: () => require('../extensions/scratch3_makeymakey'),
+    pigpio: () => require('../extensions/scratch3_pigpio'),
+    pisensehat: () => require('../extensions/scratch3_pisensehat'),
+    pibuildhat: () => require('../extensions/scratch3_pibuildhat'),
+    pivsgpio: () => require('../extensions/scratch3_pivsgpio'),
     boost: () => require('../extensions/scratch3_boost'),
     gdxfor: () => require('../extensions/scratch3_gdx_for')
 };
