diff --git a/tmp/tmpose2scratch/install.sh b/tmp/tmpose2scratch/install.sh
index a761f5c..0aa16d3 100644
--- a/tmp/tmpose2scratch/install.sh
+++ b/tmp/tmpose2scratch/install.sh
@@ -6,18 +6,19 @@ EXTENSION_NAME=TMPose2Scratch
 EXTENSION_ID=tmpose2scratch
 COLLABORATOR="Tsukurusha, YengawaLab and Google"
 EXTENSION_DESCRIPTION="Recognize your own poses."
+EXTENSION_DIR=tmp/${EXTENSION_ID}
 
 mkdir -p node_modules/scratch-vm/src/extensions/scratch3_${EXTENSION_ID}
-cp ${EXTENSION_ID}/scratch-vm/src/extensions/scratch3_${EXTENSION_ID}/index.js node_modules/scratch-vm/src/extensions/scratch3_${EXTENSION_ID}/
+cp ${EXTENSION_DIR}/scratch-vm/src/extensions/scratch3_${EXTENSION_ID}/index.js node_modules/scratch-vm/src/extensions/scratch3_${EXTENSION_ID}/
 mv node_modules/scratch-vm/src/extension-support/extension-manager.js node_modules/scratch-vm/src/extension-support/extension-manager.js_orig
 sed -e "s|class ExtensionManager {$|builtinExtensions['${EXTENSION_ID}'] = () => require('../extensions/scratch3_${EXTENSION_ID}');${LF}${LF}class ExtensionManager {|g" node_modules/scratch-vm/src/extension-support/extension-manager.js_orig > node_modules/scratch-vm/src/extension-support/extension-manager.js
 
-mkdir -p src/lib/libraries/extensions/${EXTENSION_ID}
-cp ${EXTENSION_ID}/scratch-gui/src/lib/libraries/extensions/${EXTENSION_ID}/${EXTENSION_ID}.png src/lib/libraries/extensions/${EXTENSION_ID}/
-cp ${EXTENSION_ID}/scratch-gui/src/lib/libraries/extensions/${EXTENSION_ID}/${EXTENSION_ID}-small.png src/lib/libraries/extensions/${EXTENSION_ID}/
-mv src/lib/libraries/extensions/index.jsx src/lib/libraries/extensions/index.jsx_orig
-mv src/containers/extension-library.jsx src/containers/extension-library.jsx_orig
-cp ${EXTENSION_ID}/scratch-gui/src/containers/extension-library.jsx src/containers/extension-library.jsx
+mkdir -p node_modules/scratch-gui/src/lib/libraries/extensions/${EXTENSION_ID}
+cp ${EXTENSION_DIR}/scratch-gui/src/lib/libraries/extensions/${EXTENSION_ID}/${EXTENSION_ID}.png node_modules/scratch-gui/src/lib/libraries/extensions/${EXTENSION_ID}/
+cp ${EXTENSION_DIR}/scratch-gui/src/lib/libraries/extensions/${EXTENSION_ID}/${EXTENSION_ID}-small.png node_modules/scratch-gui/src/lib/libraries/extensions/${EXTENSION_ID}/
+mv node_modules/scratch-gui/src/lib/libraries/extensions/index.jsx node_modules/scratch-gui/src/lib/libraries/extensions/index.jsx_orig
+mv node_modules/scratch-gui/src/containers/extension-library.jsx node_modules/scratch-gui/src/containers/extension-library.jsx_orig
+cp ${EXTENSION_DIR}/scratch-gui/src/containers/extension-library.jsx node_modules/scratch-gui/src/containers/extension-library.jsx
 DESCRIPTION="\
     {${LF}\
         name: '${EXTENSION_NAME}',${LF}\
@@ -51,4 +52,4 @@ DESCRIPTION="\
             }${LF}\
         }${LF}\
     },"
-sed -e "s|^export default \[$|import ${EXTENSION_ID}IconURL from './${EXTENSION_ID}/${EXTENSION_ID}.png';${LF}import ${EXTENSION_ID}InsetIconURL from './${EXTENSION_ID}/${EXTENSION_ID}-small.png';${LF}${LF}export default [${LF}${DESCRIPTION}|g" src/lib/libraries/extensions/index.jsx_orig > src/lib/libraries/extensions/index.jsx
+sed -e "s|^export default \[$|import ${EXTENSION_ID}IconURL from './${EXTENSION_ID}/${EXTENSION_ID}.png';${LF}import ${EXTENSION_ID}InsetIconURL from './${EXTENSION_ID}/${EXTENSION_ID}-small.png';${LF}${LF}export default [${LF}${DESCRIPTION}|g" node_modules/scratch-gui/src/lib/libraries/extensions/index.jsx_orig > node_modules/scratch-gui/src/lib/libraries/extensions/index.jsx
